tests/keras/applications/applications_test.py

```diff
@@ -8,6 +8,27 @@
 from keras import backend as K
 
 
+def clean_run(model_fn):
+    if K.backend() == 'cntk':
+        # Create model in a subprocess so that the memory consumed by InceptionResNetV2 will be
+        # released back to the system after this test (to deal with OOM error on CNTK backend)
+        # TODO: remove the use of multiprocessing from these tests once a memory clearing mechanism
+        # is implemented in the CNTK backend
+        def target(queue):
+            model = model_fn()
+            queue.put(model)
```

Comment:
I don't think we can pickle Keras models back to the main process. That's why only the output shape is put into the queue in the original lines.

Reply:
Updated. Thank you for the comments @myutwo150.